packages/panels/resources/views/components/sidebar/item.blade.php

```diff
@@ -48,7 +49,7 @@
         @endif
         class="fi-sidebar-item-btn"
     >
-        @if (filled($icon) && ((! $subGrouped) || $sidebarCollapsible))
+        @if (filled($icon) && ((! $subGrouped) || $sidebarCollapsible) && (! $subNavigation))
```

Comment:
Just checking, are you sure this should not be like this?
```suggestion
        @if (filled($icon) && ((! $subGrouped) || ($sidebarCollapsible && (! $subNavigation))))
```

So that when `$subGrouped` is false it does not matter if `$subNavigation` is true or false?

Does my suggestion work?